protocol/activity_center.go

```diff
@@ -29,6 +29,11 @@ const (
 	ActivityCenterNotificationTypeContactVerification
 	ActivityCenterNotificationTypeContactRemoved
 	ActivityCenterNotificationTypeNewKeypairAddedToPairedDevice
+	ActivityCenterNotificationTypeOwnerTokenReceived
+	ActivityCenterNotificationTypeSetSignerSuccess
+	ActivityCenterNotificationTypeSetSignerFailed
+	ActivityCenterNotificationTypeSetSignerDeclined
+	ActivityCenterNotificationTypeLostOwnership
```

Comment:
Don't we need yet another notification type -> `ActivityCenterNotificationTypeBecameControlNode`? (ActivityCenterNotificationTypeSetSignerSuccess != ActivityCenterNotificationTypeBecameControlNode)

https://www.figma.com/file/qHfFm7C9LwtXpfdbxssCK3/Kuba%E2%8E%9CDesktop---Communities?type=design&node-id=37206-87325&mode=design&t=Klfu2omdUgTGSnWN-0

Reply:
Actually, I can rename ActivityCenterNotificationTypeSetSignerSuccess to ActivityCenterNotificationTypeBecameControlNode or ActivityCenterNotificationTypeReceivedOwnership.
Because we do not need these two in AC, but only one in the end of the whole process (promoting to control node happens just after setting signer).